app/views/providers/email_addresses/show.html.erb

```diff
@@ -0,0 +1,26 @@
+<%= page_template(
+      page_title: t('.page_title'),
+      page_heading_options: { margin_bottom: 3 }
+    ) do %>
+
+  <%= form_with(
+        model: @form,
+        url: providers_legal_aid_application_email_address_path,
+        method: :patch,
+        local: true
+      ) do |form| %>
+
+    <div class="govuk-!-padding-bottom-6"></div>
+
+    <%= form.govuk_text_field(
+          :email,
+          label: { size: :m },
+          class: 'govuk-input govuk-!-width-three-quarters'
```

Comment:
you shouldn't need to add `govuk-input` here as `govuk_text_field` already adds it by default